index-metagenomics.md

```diff
@@ -122,6 +125,20 @@ The tools used in the EBI Metagenomics pipeline are also available here and can
 
 However, the parameters must be adjusted by the user as we could not find them in the EBI Metagenomics documentation.
 
+## ASaiM-MT: Optimized workflow for metatranscriptomics data analysis
+While the shotgun workflow is suitable for both metagenomics and metatranscriptomics datasets, we also offer an enhanced workflow aimed specifically at metatranscriptomics data.
+
+![](https://asaim.readthedocs.io/en/latest/_images/asaim-mt_workflow.png)
```

Comment:
maybe we should duplicate the images. Not sure if anyone that changes the ASAIM website takes this page into account as well.

Reply:
I dont think anybody besides @bebatut and me change ASaiM :P but am happy to add the image here as well to be safe indeed